oak/server/BUILD

```diff
@@ -65,6 +65,16 @@ cc_library(
     ],
 )
 
+cc_binary(
```

Comment:
Is there a way to indicate a fuzz corpus in Bazel-land?  If there is, we could start with the oak/server/testdata/*.wasm genfiles to guide the fuzzer towards inputs that aren't just going to fail at the very first `if` test.